README.md

```diff
@@ -1113,10 +1113,20 @@ Does not support the following attributes: [link](https://www.cisco.com/c/en/us/
 
 #### port_channel
 
+##### 2960
+
+##### 3560
+
+##### 3650
+
 ##### 3750
 
+##### 4503
+
 ##### 4507
 
+##### 4948
+
 This device does not have native trunking. It does not support the following attributes: [link](https://learningnetwork.cisco.com/thread/75947)
```

Comment:
This line looks like it belongs to the 4507 in the original readme? Should it be moved up?

Reply:
It's shared by all of them. Thought this was better than repeating it 10 times.

Reply:
OK, then reword this to: 
"The above devices do not have native trunking. The following attributes are not supported : [link](https://learningnetwork.cisco.com/thread/75947)"

Reply:
Updated